apps/io-wallet-user-func/local.settings.json.example

```diff
@@ -29,6 +29,7 @@
     "EntityConfigurationStorageAccount__serviceUri": "ENTITY_CONFIGURATION_STORAGE_ACCOUNT__SERVICE_URI",
     "EntityConfigurationStorageContainerName": "ENTITY_CONFIGURATION_STORAGE_CONTAINER_NAME",
     "StorageConnectionString": "STORAGE_ACCOUNT_CONNECTION_STRING",
+    "CreationQueueName": "WALLET_INSTANCE_CREATION_QUEUE_NAME",
```

Comment:
specify creation of the wallet instance in the queue name env

Reply:
I thought about it! Can I do the same for `RevocationQueueName`?